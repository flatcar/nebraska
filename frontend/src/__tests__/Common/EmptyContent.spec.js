import { render } from '@testing-library/react';
import Empty from '../../components/common/EmptyContent';

describe('Empty', () => {
  it('renders correct content', () => {
    const { asFragment } = render(<Empty>Some Dummy Content</Empty>);
    expect(asFragment()).toMatchSnapshot();
  });
});
