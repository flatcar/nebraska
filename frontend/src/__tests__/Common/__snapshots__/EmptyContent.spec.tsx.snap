// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Empty > renders correct content 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-19midj6"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter css-11r2klu-MuiTypography-root"
      data-testid="empty"
      style="--Typography-textAlign: center;"
    >
      Some Dummy Content
    </p>
  </div>
</DocumentFragment>
`;
